src/create_disk.sh

```diff
@@ -67,14 +68,25 @@ else
 	ext="AA64"
 fi
 
+# ensure that home is created with the correct SELinux context
+realvar=$(readlink -f rootfs/ostree/deploy/*/var)
+mkdir -p "${realvar}/home"
+setfattr -n "security.selinux system_u:object_r:home_root_t:so" "${realvar}/home"
+
+# install bios grub
+grub2-install \
+	--target i386-pc \
+	--boot-directory rootfs/boot \
+	$disk
+
 # install uefi grub
-mkdir -p rootfs/boot/efi/EFI/{BOOT,fedora}
-cp "/boot/efi/EFI/BOOT/BOOT${ext}.EFI" "rootfs/boot/efi/EFI/BOOT/BOOT${ext}.EFI"
-cp "/boot/efi/EFI/fedora/grub${ext,,}.efi" "rootfs/boot/efi/EFI/BOOT/grub${ext,,}.efi"
-cat > rootfs/boot/efi/EFI/fedora/grub.cfg << 'EOF'
-search --label boot --set prefix
-set prefix=($prefix)/grub2
-normal
+mkdir -p rootfs/boot/efi/EFI/{BOOT,"${os_name}"}
+find /boot/efi/EFI/BOOT/ -type f -iname "*efi" -exec cp -v {} rootfs/boot/efi/EFI/BOOT/ \;
+find /boot/efi/EFI/"${os_name}" -type f -iname "*efi" -exec cp -v {} rootfs/boot/efi/EFI/BOOT/ \;
+chown -R 0:0 rootfs/boot/efi/EFI
+cat > rootfs/boot/efi/EFI/"${os_name}"/grub.cfg << EOF
+set prefix="(hd0,gpt1)/grub2"
+source "(hd0,gpt1)/grub2/grub.cfg"
```

Comment:
Why not use the prefix you just set?

Reply:
Check out what master does now. Should just be `normal` since the `normal` command sources `$prefix/grub.cfg` anyway

Reply:
I thought I rebased correctly, but didn't....I'll get this fixed and tested shortly.